hazelcast/src/main/java/com/hazelcast/sql/SqlColumnMetadata.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+
+/**
+ * SQL column metadata.
+ */
+public class SqlColumnMetadata implements Serializable {
+
+    private static final long serialVersionUID = 1522422443380097801L;
+
+    private final String name;
+    private final SqlColumnType type;
+
+    public SqlColumnMetadata(String name, SqlColumnType type) {
+        assert name != null;
+        assert type != null;
+
+        this.name = name;
+        this.type = type;
+    }
+
+    /**
+     * Get column name.
+     *
+     * @return Column name.
+     */
+    @Nonnull
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Gets column type.
+     *
+     * @return Column type.
+     */
+    @Nonnull
+    public SqlColumnType getType() {
+        return type;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        SqlColumnMetadata that = (SqlColumnMetadata) o;
+
+        if (!name.equals(that.name)) {
+            return false;
+        }
+
+        return type == that.type;
+    }
+
+    @Override
+    public int hashCode() {
+        int result = name.hashCode();
+
+        result = 31 * result + type.hashCode();
+
+        return result;
+    }
+
+    @Override
+    public String toString() {
+        return '[' + name + ':' + type + ']';
```

Comment:
We could use the sql syntax `name + ' ' + type`. Also the `[]` suggest a collection, but it's a tuple, i'd just use none.

Reply:
Fixed.